mock/mock.go

```diff
@@ -252,18 +252,16 @@ func (m *Mock) findClosestCall(method string, arguments ...interface{}) (bool, *
 	return true, closestCall
 }
 
-func callString(method string, arguments Arguments, includeArgumentValues bool) string {
-
-	var argValsString string
-	if includeArgumentValues {
-		var argVals []string
-		for argIndex, arg := range arguments {
-			argVals = append(argVals, fmt.Sprintf("%d: %#v", argIndex, arg))
+func callString(method string, arguments Arguments) string {
+	var argVals []string
+	for i, arg := range arguments {
+		if i > 0 {
+			argVals = append(argVals, ",")
 		}
-		argValsString = fmt.Sprintf("\n\t\t%s", strings.Join(argVals, "\n\t\t"))
+		argVals = append(argVals, fmt.Sprintf("%#v", arg))
```

Comment:
We should:
 - **Use `%s` instead of `%#v`**: it is pretty common methods to receive `[]byte` and `%#v` would print output like `[]byte{0x6f, 0x6b}`
 - **Split arguments in different lines**: it is common for people to receive big structs or long strings. Having all the arguments in a single line will affect readability.